app/components/ui/tale-tabs-selector/template.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="ui">
+  <div class="ui secondary pointing menu" style="display:inline-flex;">
+      <a class="item {{if activeTabInteract "active"}}" data-tab="tab-interact" onclick={{action "activateInteract"}}>Interact</a>
+      <!--<a class="item {{if activeTabFiles "active"}}" data-tab="tab-files" onclick={{action "activateFiles"}}>Files</a>-->
```

Comment:
Comment in `.hbs` files should be made with {{!-- --}}, otherwise there are instances where things break when they are compiled to `.js` files by Ember

Reply:
Replaced `<!-- -->` with `{{!-- --}}` here.. I think that's my only comment in a template for now...

I went ahead and stubbed the component here for the new Data tab in hopes that it would prevent some future merge conflict.